lib/node_loader.js

```diff
@@ -45,6 +45,7 @@ require('./http/node');
 require('./shared-ini/ini-loader');
 
 // Load custom credential providers
+require('./credentials/assume_role_with_web_identity_credentials');
```

Comment:
reminder: don't forget to add this credential provider to Node's default credential provider chain. It will be placed before [EC2 credental provider](https://github.com/aws/aws-sdk-js/blob/2627d1af01bf1a1dc745361c3edf9e825594dcd4/lib/node_loader.js#L66)

Reply:
Will do :+1:

Reply:
Resolved in https://github.com/aws/aws-sdk-js/pull/2737/commits/9d2019cdc65d38961e199a62e4507ecec16ef531